htmresearch/frameworks/nlp/classify_htm.py

```diff
@@ -84,6 +84,10 @@ def __init__(self,
       self.networkConfig["classifierRegionConfig"].get("regionName")]
 
 
+  def getKNN(self):
+    return self.classifierRegion.getSelf()._knn
```

Comment:
Instead of accessing the private member directly, it should call the region's getAlgorithmInstance() method to get the instance. i.e.something like:

```
self.classifierRegion.getSelf().getAlgorithmInstance()
```